Example/Emission/AppDelegate.m

```diff
@@ -88,12 +90,20 @@ - (void)setupEmission;
 
   emission.switchBoardModule.presentNavigationViewController = ^(UIViewController * _Nonnull fromViewController,
                                                                  NSString * _Nonnull route) {
+    if (self.isShowingStorybook) {
+      NSLog(@"Route push - %@", route);
+      return;
+    }
     [fromViewController.navigationController pushViewController:[self viewControllerForRoute:route]
                                                        animated:YES];
   };
 
   emission.switchBoardModule.presentModalViewController = ^(UIViewController * _Nonnull fromViewController,
                                                             NSString * _Nonnull route) {
+    if (self.isShowingStorybook) {
+      NSLog(@"Route modal - %@", route);
+      return;
+    }
```

Comment:
I experimented with trying to pass a `console.log` back to JS world, but it just wasn't working very well.